packages/contracts/src/dollar/mocks/MockBondingShareV2.sol

```diff
@@ -0,0 +1,226 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.3;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC1155/extensions/ERC1155Burnable.sol";
+import "@openzeppelin/contracts/token/ERC1155/extensions/ERC1155Pausable.sol";
+import "../core/UbiquityDollarManager.sol";
+import "../utils/SafeAddArray.sol";
+
+contract BondingShareV2 is ERC1155, ERC1155Burnable, ERC1155Pausable {
```

Comment:
We renamed this to Staking
```suggestion
contract Staking is ERC1155, ERC1155Burnable, ERC1155Pausable {
```

Reply:
This script is interacting with contracts as currently deployed. If you try to call Staking.function() on mainnet it'll revert.